### ISSUE TITLE
Increment overflow

### ISSUE THREAD

**Issue by moander** [googleapis/nodejs-bigtable#103]
I tried to increment a value by Number.MAX_SAFE_INTEGER 

Incrementing beyond int64 gives you a string in return:
```js
      "hulahoi": [
        {
          "value": "\u0000 \u0000\u0000\u0000\u0000\u0000\u0001",
          "labels": [],
          "timestamp": "1524579337814000"
        }
      ],

```

#### Environment details

OS: macos 10.13.3
Node.js version: 8.10.0
npm version: 5.6.0
yarn version: 1.3.2
@google-cloud/bigtable version: master branch (0.13)

#### Steps to reproduce

```js
    await table.row('gwashington').increment('fam1:hulahoi');
    await table.row('gwashington').increment('fam1:hulahoi', Number.MAX_SAFE_INTEGER);
```

**Comment by sduskis** [comment]
@kolodny: is this bug actionable?

**Comment by kolodny** [comment]
This is actually fixed via #156
